vsintegration/Vsix/VisualFSharpFull/VisualFSharpFull.csproj

```diff
@@ -189,6 +189,7 @@
 
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonPackageVersion)" />
+    <PackageReference Include="Microsoft.Diagnostics.Runtime" Version="0.9.180305.1" />
```

Comment:
This version number should be added to `build\targets\PackageVersions.props` under the `other packages` heading and placed in alphabetical order:

``` xml
<MicrosoftDiagnosticsRuntimePackageVersion>0.9.180305.1</MicrosoftDiagnosticsRuntimePackageVersion>
```

And replaced in this file with `$(MicrosoftDiagnosticsRuntimePackageVersion)`.